### ISSUE TITLE
Profile Image is not properly aligned in the Theme after recent update.

### ISSUE THREAD

**Issue by AlizarinRed** [Comfy-Themes/Discord#149]
![image](https://github.com/Comfy-Themes/Discord/assets/31637936/240818d7-573e-438d-9069-b9ab80db03c8)
Image should help give context.

I'm using Vencord on the most recent version.

**Comment by ffuentesm** [comment]
are you using another theme or something that changes the layout? 
From what I see in your image, you have something that makes the server groups open sideways instead of downward. 
That uses the text layout to shift to the right while your image icon stays in the same place since it's in a different layer. 

I had the opposite problem as I ma using a theme that removed my server list from the left side. This causes the layout to shift to the left, which causes my icon to be on the right side. 
![image](https://github.com/Comfy-Themes/Discord/assets/44974827/cf8c0fe8-4bf1-4dd9-9ba6-45c59e2b1c7e)

A way to solve this is to edit the pixel value of the spacing of the profile image layout

**Comment by AlizarinRed** [comment]
Nope, currently it's the only theme I have. If I were to ponder a guess. I think it might be because it's using a form of % adjustments to it that don't respect window size properly as my monitor is an Ultra Wide Screen monitor.
I do have the Better Folders Plugin that adds an extra line on the left of folders that's opened.

When I disabled that plugin it was in it's proper place. So potentially having the theme account for that might be a good idea.
